OneCommunityGlobal/HighestGoodNetworkApp
# Yi fix minor formatting and styling issues

# Description
<img width="708" alt="Screenshot 2023-11-18 at 3 26 57 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/6d5159e2-d52e-4c85-86f2-5639d19933ea">

https://www.loom.com/share/3eb48d69e36249ef945a709197713ced?sid=1571d129-fd39-4c72-a5cb-ad870b7bbc7d

## Related PRS (if any):
none

## Main changes explained:
- Change the information tooltip position to the center of the page.
- Align certain items to be on the same line at the top of the page, correcting a bump-down issue.
- In individual profiles, fix a spacing issue to ensure uniformity.
- Adjust the spacing so each element has the same amount of space.
- Make phone input width 100% so they match other input length.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. verify information tooltip on center
5. verify information tooltip, active icon, and use icon on same line
6. verify information tooltip on user profile page spacing fixed
7. verify phone input on user profile page have same width as other input

## Screenshots or videos of changes:

## Note:
Before change:
https://www.loom.com/share/3eb48d69e36249ef945a709197713ced?sid=1571d129-fd39-4c72-a5cb-ad870b7bbc7d
After change:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/ad000f8f-2291-462c-b727-66cf742d957a
